packages/aila/src/protocol/parseMessageRow.test.ts

```diff
@@ -0,0 +1,67 @@
+import invariant from "tiny-invariant";
```

Comment:
This is a test for the bug that I was working on today.

Basically, the Zod schema was too relaxed, and if you do [ZodSchema].safeParse it would return data = {} instead of data = {a validated cycle}.

Very strange and unexpected behavour. I've left a comment on the relevant type that was causing the problem, but I've had to leave it in place.

Reply:
I think it could be useful context to put this in the test file itself?